app/src/main/java/org/oppia/app/topic/train/SkillSelectionAdapter.kt

```diff
@@ -0,0 +1,53 @@
+package org.oppia.app.topic.train;
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.databinding.DataBindingUtil
+import androidx.databinding.ViewDataBinding
+import androidx.databinding.library.baseAdapters.BR
+import androidx.recyclerview.widget.RecyclerView
+import kotlinx.android.synthetic.main.topic_train_skill_view.view.*
+import org.oppia.app.R
+import org.oppia.app.databinding.TopicTrainSkillViewBinding
+
+// TODO(#172): Make use of generic data-binding-enabled RecyclerView adapter.
+/** Adapter to bind skills to [RecyclerView] inside [TopicTrainFragment]. **/
```

Comment:
Correct this java doc comment. Must ends with */

Reply:
Thanks for this nit. Done.